src/components/dropdown/dropdown.js

```diff
@@ -87,4 +109,4 @@ class Dropdown extends Component {
   }
 }
 
-export default WithMenuContext(Dropdown);
```

Comment:
Do you propose not to use context for active element implementation? Do you have another idea?

Reply:
This solution caused a lot of tests failures. we didn't see it since, we didn't have a build. Now we have. Anyway, I will make it work during the day. 
For now, I want this code inside, to not stop hadas work

Reply:
:+1: